test/functionalTest/cases/3071_DELETE_with_wrong_servicepath/3071_DELETE_operation_with_wrong_servicepath.test

```diff
@@ -0,0 +1,189 @@
+# Copyright 2021 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+SP with wildcard (/#) and multiple service-path are not allowed in DELETE operation
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+# 
+# 01. Create entity E1/T1/A1=1, with servicepath '/Level_1/'
+# 02. GET /v2/entities/E1, with servicepath '/Level_1/' to see E1/T1/A1=1
+# 03. Create entity E2/T2/A2=2, with servicepath 'Level_2/'
+# 04. Get /v2/entities/E2, with servicepath '/Level_2/' to see E2/T2/A2=2
+# 05. Attempt to DELETE entity E1, using DELETE operation, with servicepath '/#', see 400 error
+# 06. Attempt to DELETE entity E2, using DELETE operation, with servicepath '/Level_1/, /Level_2/' see 400 error
+#
+
+
+echo "01. Create entity E1/T1/A1=1, with servicepath '/Level_1/'"
+echo "=========================================================="
+payload='{
+    "id": "E1",
+    "type": "T1",
+    "A1": {
+      "value": 1,
+      "type": "Integer"
+    }
+}'
+orionCurl --url /v2/entities --payload "$payload" --header "Fiware-ServicePath:/Level_1/"
+echo
+echo
+
+
+echo "02. GET /v2/entities/E1 with servicepath '/Level_1/' to see E1/T1/A1=1"
+echo "======================================================================"
+orionCurl --url /v2/entities/E1 --header "Fiware-ServicePath:/Level_1/"
```

Comment:
Use `--servicePath` instead of `--header` in every case. You can have a look to existing .test to see how to use `--servicePath`

Reply:
Fixed in cdab795